packages/perseus-editor/src/__stories__/editor.stories.tsx

```diff
@@ -1,17 +1,21 @@
-import {ApiOptions} from "@khanacademy/perseus";
+/* eslint-disable react/prop-types */
```

Comment:
Just curious, why was this needed?

Reply:
ESLint's `prop-types` rule doesn't understand that there _is_ prop type validation the `onChange` callback of the `<Editor />` on line 88.  But, we don't really need prop-types validation because we have Typescript. 

See: https://github.com/Khan/perseus/pull/538